documentation/staging/content/faq/newbie.md

```diff
@@ -0,0 +1,139 @@
+
+***
+
+#### Load Balancers
+
+**Q:** Load balancing and failover inside a DataCenter (HTTPS and T3s)?
+
+**A:** We originally certified on Traefik with the Kubernetes cluster; this is a very basic load balancer.  We have also certified other more sophisticated load balancers. See [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}).
+
+***
+
+#### Life Cycle and Scaling
+
+**Q:** How to deal with grow and shrink? Cluster and non-cluster mode.
+
+**A:** You can scale and shrink a configured WebLogic cluster (a set of preconfigured Managed Servers) or a dynamic WebLogic cluster (a cluster that uses templated Managed Servers) using different methods. See the blog, [Automatic Scaling of WebLogic Clusters on Kubernetes](https://blogs.oracle.com/weblogicserver/automatic-scaling-of-weblogic-clusters-on-kubernetes-v2).
+* Manually, using Kubernetes command-line interface, `kubectl`.
+* WLDF rules and policies; when the rule is met the Administration Server sends a REST call to the operator which calls the Kubernetes API to start a new pod/container/server.
+* We have developed and made open source the [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter) which exports WebLogic metrics to Prometheus and Grafana.  In Prometheus, you can set rules similar to WLDF and when these rules are met, a REST call is made to the operator which invokes a Kubernetes API to start a new pod. See to blog, [Announcing the New WebLogic Monitoring Exporter](https://blogs.oracle.com/weblogicserver/announcing-the-new-weblogic-monitoring-exporter-v2).
+
+
+**Q:** Container life cycle: How to properly spin up and gracefully shut down WLS in a container?
+
+**A:** The operator manages container/pod/WebLogic Server life cycle automatically; it uses the Node Manager (internally) and additional approaches  to do the following operations:
+* Entrypoint - start WebLogic Server.
+* Liveliness probe – check if the WebLogic Server is alive.
+* Readiness probe – query if the WebLogic Server is ready to receive requests.
+* Shutdown Hook – gracefully shut down WebLogic Server.
+
+These operations also can be done manually using the Kubernetes command-line interface, `kubectl`.
+
+***
+
+
+#### Patching and Upgrades
+
+**Q:** Patching: rolling upgrades, handling of one-off-patches and overlays, CPUs, and such.
+
+**A:** Patches are applied using OPatch and rolled out in the following fashion:
+* Upgrade or patch a WebLogic cluster in a Docker environment.
+* Apply PSUs for important security upgrades.
+* In Dockerfile, extend the image and apply patch using OPatch.
+* Operator starts rolling upgrade calling Kubernetes API.
+Starting the pods/containers/servers in a rolling fashion can be invoked manually, calling the Kubernetes API directly.  See the blog, [Patching WebLogic Server in a Kubernetes Environment](https://blogs.oracle.com/weblogicserver/patching-weblogic-server-in-a-kubernetes-environment).
```

Comment:
This "See" reference is out of date. Instead, reference the new patching chapter, the life-cycle doc, and the CI/CD doc...  I assume @mriccell or @rjeberhard may also want to word-smith the contents of the answer.

Reply:
@tbarnes-us I can't x-ref the new patching section until it's in the docs, but I added the other two. If @mriccell or @rjeberhard approves the patching doc, I'll add it.

Reply:
As above, I'm tempted to remove the question or at least the content here and just point to the new patching documentation. Which one do I need to approve (overwhelmed today)?

Reply:
@rjeberhard https://github.com/oracle/weblogic-kubernetes-operator/pull/2328
(Sorry for the overload.)